logicrunner/executionbroker.go

```diff
@@ -45,7 +46,7 @@ type ExecutionBrokerI interface {
 	PendingState() insolar.PendingState
 	PrevExecutorStillExecuting(ctx context.Context)
 	PrevExecutorPendingResult(ctx context.Context, prevExecState insolar.PendingState)
-	PrevExecutorFinishedPending(ctx context.Context) error
+	PrevExecutorPendingFinished(ctx context.Context) error
```

Comment:
previous name was valid english, new name is wrong :) Either use old or PrevExecutorSentPendingIsFinished.

Reply:
`PrevExecutorPendingIsFinished` is fine too.